### PR TITLE
Script to update gh-pages in CI

### PR DESCRIPTION
**Pull request by lbonn**

### CODE REVIEW

**Comment by patrickvacek** [comment]
Looks pretty good to me!

**Comment by lbonn** [comment]
After a few unsuccessful tries, it's now working.

File: src/sota_tools/CMakeLists.txt [hunk 1]
```diff
@@ -181,7 +181,7 @@ if (BUILD_SOTA_TOOLS)
     # Get targets.json from images repository.
     # Find specified OSTree ref in targets.json.
     add_test(NAME check_not_expired_targets
-        COMMAND ${PROJECT_SOURCE_DIR}/tests/run_expired_test.sh 2019-11-17T23:58:40Z ${CMAKE_BINARY_DIR}
+        COMMAND ${PROJECT_SOURCE_DIR}/tests/run_expired_test.sh 2031-11-17T23:58:40Z ${CMAKE_BINARY_DIR}
```
Comment (patrickvacek) [inline src/sota_tools/CMakeLists.txt]: Lol nice.

**Comment by patrickvacek** [comment]
Appears to have worked nicely: https://main.gitlab.in.here.com/olp/edge/ota/connect/client/aktualizr/-/jobs/835026

**Comment by patrickvacek** [comment]
Well, better proof perhaps: https://advancedtelematic.github.io/aktualizr/2019.10/index.html

**Comment by patrickvacek** [comment]
Actually, one more request when you get the chance: can you update the docs (https://docs.ota.here.com/ota-client/latest/release-process.html#_update_doxygen_on_github)? We can keep the old full description around for reference, but we can say now that it's automated.

**Comment by lbonn** [comment]
Ok, I think I'll reword it and remove most of the details to refer to the script. The shorter these instructions are, the more likely people will follow them until the end :)

**Pull request merged**